pkg/planner/cardinality/selectivity_test.go

```diff
@@ -1085,6 +1085,44 @@ func TestOrderingIdxSelectivityThreshold(t *testing.T) {
 	}
 }
 
+func TestOrderingIdxSelectivityRatio(t *testing.T) {
+	store, dom := testkit.CreateMockStoreAndDomain(t)
+	testKit := testkit.NewTestKit(t, store)
+	h := dom.StatsHandle()
+
+	testKit.MustExec("use test")
+	testKit.MustExec("drop table if exists t")
+	testKit.MustExec("create table t(a int primary key, b int, c int, index ib(b), index ic(c))")
+	for i := 0; i < 10000; i++ {
+		testKit.MustExec(fmt.Sprintf("insert into t values (%v, %v, %v)", i, i, i)) // [0, 10000)
```

Comment:
I updated the testcase to use 20 insert statements rather than 10,000. I also only insert 1000 rows, since this is sufficient for the test, and reduces analyze cost also. There is another testcase in this file that currently has 400 inserts, and the test for this PR was modeled from that insert example.

Reply:
I updated the test to use mock statistics rather than inserts + analyze. This should speed up the testcase.